packages/core/test/__snapshots__/BlockLink.js.snap

```diff
@@ -5,23 +5,19 @@ exports[`BlockLink renders 1`] = `
   cursor: pointer;
   -webkit-text-decoration: none;
   text-decoration: none;
-  color: #007aff;
```

Comment:
⁉️

Reply:
Not sure exactly why this changed, I'll have to look into it.

Reply:
@sdalonzo If I'm reading this correctly, `color: inherit;` at line 21 would have already been overriding this color line, so this deletion shouldn't have any real impact. I verified `Link` and `BlockLink` again in Storybook and everything looks as expected.